frontend/src/constants/routes.ts

```diff
@@ -1,7 +1,7 @@
 export enum ROUTES {
     Home = "/",
     Products = "/products",
-    ProductDetails = "/products/:id",
```

Comment:
i dont think enum for Routes is required / You can simply create a const

Reply:
i changed them back to constants dai